### PR TITLE
Release/2.2.1

### PR DESCRIPTION
**Pull request by Dmitriy1984Molchanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@ This application is supposed to be used as API for [Laboratory UI](https://githu
 ### vNext
 * ...
 
+### v2.2.1
```
Comment (AndreiNavoichyk) [inline README.md:12]: Minor (not patch) should be increased.

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@Cerebellum-Network/laboratory-api",
   "repository": "git://github.com/Cerebellum-Network/laboratory-api.git",
-  "version": "2.2.0",
+  "version": "2.2.1",
```
Comment (AndreiNavoichyk) [inline package.json:4]: Version in `package-lock.json` should be updated also.

File: README.md (hunk 1 above)
Comment (Dmitriy1984Molchanov) [reply README.md:12]: fixed

File: package.json (hunk 2 above)
Comment (Dmitriy1984Molchanov) [reply package.json:4]: fixed

**Pull request closed without merging**